conga-aem-definitions/src/main/templates/aem-dispatcher/publish/_vhost_publish_shared_include.conf.hbs

```diff
@@ -8,7 +8,8 @@ TraceEnable Off
 
 # Enable rewrite engine
 RewriteEngine On
-
+# Don't rewrite the dispatcher invalidation URL
+RewriteCond %{REQUEST_URI} !^/dispatcher/invalidate.cache
```

Comment:
this should be placed immediately before the RewriteRule directive - otherwise it is just applied on the first (and only the first) RewriteRule statement in the other config files (whatever this may be).